src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -81,18 +81,18 @@ public static Address parseAddress(String address) throws ParseException {
     }
 
     /**
-     * Parses a {@code String email} into an {@code Email}.
+     * Parses a {@code String expiryDate} into an {@code ExpiryDate}.
```

Comment:
Very minor, should be "an" instead of "a".

Reply:
Corrected.